package.json

```diff
@@ -43,6 +43,7 @@
     "bootstrap-filestyle": "~1.2.1",
     "bootstrap-switch": "3.3.4",
     "classnames": "~2.2.6",
+    "clsx": "^1.0.4",
```

Comment:
Just one line above, we pull in the `classnames` package which does the same thing as `clsx`.

Reply:
in ui-classic, we only have one use of `classnames`, which should be trivial to change.

```
app/javascript/forms/miq-button.jsx
3:import ClassNames from 'classnames';
14:  const klass = ClassNames({
```

But, in v2v, there's 28 more uses of `classnames`.

So, I'd suggest just going with `classnames`, unless you want to update ManageIQ/manageiq-v2v too.

Reply:
@fhlavac ok `classnames` it is. Can you please change it? The package has almost the same syntax as clsx. You can find it in the docs.